bin/run-browser-test-env

```diff
@@ -12,45 +12,73 @@ if [ -f ".secrets" ]; then
   . .secrets
 fi
 
+ci="no"
+
 # Get cloud provider
 _setArgs(){
   while [ "${1:-}" != "" ]; do
     case "$1" in
+      "--ci")
+        ci="yes"
+        ;;
+
       "--azure")
-       cloudProvider="azure"
-       emulator="azurite"
-       devServer="http://localhost:10000"
-       export STORAGE_SERVICE_NAME="azure-blob"
-       ;;
+        readonly CLOUD_PROVIDER="azure"
+        readonly EMULATOR="azurite"
+        readonly SERVER_URL="http://localhost:10000"
+        export STORAGE_SERVICE_NAME="azure-blob"
+        ;;
+
       "--aws")
-       cloudProvider="aws"
-       emulator="localstack"
-       devServer="http://localhost:6645"
-       export STORAGE_SERVICE_NAME="s3"
+        readonly CLOUD_PROVIDER="aws"
+        readonly EMULATOR="localstack"
+        readonly SERVER_URL="http://localhost:6645"
+        export STORAGE_SERVICE_NAME="s3"
        ;;
     esac
+
     shift
   done
 }
 
 _setArgs "$@"
 
 # default to AWS for now
-if [ -z "$cloudProvider" ]; then
-  cloudProvider="aws" && emulator="localstack" && devServer="http://localhost:6645"
+if [ -z "${CLOUD_PROVIDER}" ]; then
+  readonly CLOUD_PROVIDER="aws"
+  readonly EMULATOR="localstack"
+  readonly SERVER_URL="http://localhost:6645"
+
   export STORAGE_SERVICE_NAME="s3"
 fi
 
+compose_command="docker-compose --profile ${CLOUD_PROVIDER}"
+compose_command="${compose_command} -f browser-test/browser-test-compose.yml"
+
+if [[ "${ci}" = "no" ]]; then
+  compose_command="${compose_command} -f browser-test/browser-test-compose.dev.yml"
```

Comment:
don't attempt to use the local sbt cache directories in CI

Reply:
Put these informative comments in the file :)

It's also unclear to me how this file accomplishes that goal, maybe comment that too.

Reply:
done